src/image-featurizer/src/main/scala/ImageFeaturizer.txt

```diff
@@ -0,0 +1,17 @@
+The ``ImageFeaturizer`` relies on a ``CNTKModel`` to do the featurization of
```

Comment:
I see ``ClassName`` used in some places and ClassName (without the back ticks) used in others - could you please homogenize?